src/Redux/actionCreators/ServiceActions.js

```diff
@@ -1,7 +1,8 @@
 import { Root } from "protobufjs";
 
-import { APIEndpoints } from "../../utility/constants/APIEndpoints";
+import { APIEndpoints } from "../../config/APIEndpoints";
 import GRPCProtoV3Spec from "../../assets/models/GRPCProtoV3Spec";
+import { Auth, API } from "aws-amplify";
```

Comment:
Move to above section of import.

Reply:
Aligned [imports](https://github.com/singnet/refactored-dapp/pull/13/commits/7a7a6b2e860778e4304122a8ed33bff0873c025b)